src/main/java/com/rapid7/net/AsyncLogger.java

```diff
@@ -555,12 +553,13 @@ void reopenConnection() throws InterruptedException {
                 }
```

Comment:
Is simply catching `IOException` sufficient above? `openConnection()` previously swallowed all exceptions.

Reply:
should be sufficient.